src/Morphir/SpringBoot/Backend.elm

```diff
@@ -414,7 +414,7 @@ mapStatefulAppImplementation opt distribution currentPackagePath currentModulePa
                                         , body =
                                             Just
                                                 (Scala.Variable
-                                                    "{statefulApp.businessLogic(state, command)}"
+                                                    "{statefulApp.businessLogic(state)(command)}"
```

Comment:
This happens to work simply because variables are pretty-printed without changes but this is not how you supposed to generate logic. You should use the building blocks in the Scala AST to compose these. This for example would be a Scala.Ref wrapped inside a Scala.Apply with state and command as variables in the argument list.

Reply:
Done